bridge/src/main/java/io/stargate/bridge/service/BridgeService.java

```diff
@@ -99,4 +106,12 @@ public void authorizeSchemaReads(
             request, GrpcService.CONNECTION_KEY.get(), authorizationService, responseObserver)
         .handle();
   }
+
+  @Override
```

Comment:
can we add a simple java docs

Reply:
The method inherits the javadoc of the generated parent class, which is in turn copied from the proto file:
```
    /**
     * <pre>
     * Checks which features are supported by the persistence backend.
     * </pre>
     */
```